docs/content/guide/i18n.ngdoc

```diff
@@ -18,8 +18,10 @@ application means providing translations and localized formats for the abstracte
 Angular supports i18n/l10n for {@link ng.filter:date date}, {@link ng.filter:number number} and
 {@link ng.filter:currency currency} filters.
 
-Additionally, Angular supports localizable pluralization support through the {@link
-ng.directive:ngPluralize `ngPluralize` directive}.
+Localizable pluralization is supported via the {@link ng.directive:ngPluralize `ngPluralize`
```

Comment:
I hadn't done that because I had mistakenly assumed that ngPluralize allowed its embedded messages to contain HTML.  Reading the docs and testing on plnkr.co shows that that is not the case.

So yes, we should definitely deprecate this for 1.5!  Should I say something about it in the docs for 1.4 right now?